tsdate/core.py

```diff
@@ -1682,7 +1790,8 @@ def variational_gamma(
     max_iterations=None,
     max_shape=None,
     match_central_moments=None,
-    global_prior=True,
+    prior_mixture_dim=1,
+    em_iterations=10,
```

Comment:
I think prior_mixture_dim and em_iteration defaults need to be None (which then get overridden below)